Sources/ImperialCore/Routing/FederatedServiceRouter.swift

```diff
@@ -77,20 +77,19 @@ extension FederatedServiceRouter {
     public var errorKey: String { "error" }
     public var callbackHeaders: HTTPHeaders { [:] }
    
-    public func configureRoutes(withAuthURL authURL: String, authenticateCallback: ((Request) throws -> (EventLoopFuture<Void>))?, on router: RoutesBuilder) throws {
-		router.get(callbackURL.pathComponents, use: callback)
-		router.get(authURL.pathComponents) { req -> EventLoopFuture<Response> in
+    public func configureRoutes(withAuthURL authURL: String, authenticateCallback: ((Request) async throws -> Void)?, on router: RoutesBuilder) async throws {
+        router.get(callbackURL.pathComponents, use: self.callback)
```

Comment:
```suggestion
        router.get(callbackURL.pathComponents, use: { try await self.callback($0) })
```
This is to remove a Sendable warning